.changeset/lazy-rockets-itch.md

```diff
@@ -0,0 +1,15 @@
+---
+'@sl-design-system/data-source': patch
+---
+
+New `@sl-design-system/data-source` package
+
+This packages provides `DataSource`, `ArrayDataSource` and `FetchDataSource` classes
+for managing data sources in the design system. At the moment, it is only used by the
+grid components, but it can be used in future components as well.
+
+`DataSource` and `ArrayDataSource` were previously part of the `@sl-design-system/shared`
+package, but they have been moved to this new package to make them more reusable.
+
```

Comment:
Maybe there should be also an information that instead of using `sl-update` event, from now should be used `sl-data-source-update`?

Reply:
I reverted the name change as discussed.